cli/cli.go

```diff
@@ -184,19 +184,20 @@ func NewApp(version, environment, hash, salt string,
 				Usage:  SetFirewallMarkUsageText,
 				Action: cmd.SetFirewallMark,
 			},
-			{
-				Name:      "ipv6",
-				Usage:     SetIpv6UsageText,
-				Action:    cmd.SetIpv6,
-				ArgsUsage: MsgSetBoolArgsUsage,
-				Description: fmt.Sprintf(
-					MsgSetBoolDescription,
-					SetIpv6UsageText,
-					"ipv6",
-					"ipv6",
-				),
-				BashComplete: cmd.SetBoolAutocomplete,
-			},
+			// TODO: uncomment once IPv6 support is enabled in the backend
```

Comment:
Will we ever have the IPv6 support? If I understood correctly, we are getting rid of this at all so maybe we don't need to comment this out and just remove?